module/metrics/execution.go

```diff
@@ -959,6 +1019,35 @@ func (ec *ExecutionCollector) RuntimeTransactionProgramsCacheHit() {
 	ec.programsCacheHit.Inc()
 }
 
+func (ec *ExecutionCollector) SetNumberOfDeployedCOAs(count uint64) {
+	ec.numberOfDeployedCOAs.Set(float64(count))
+}
+
+func (ec *ExecutionCollector) EVMTransactionExecuted(
```

Comment:
from this setup there will be no way to tell if there are more direct call failures than non-direct call failures. Is that something we need?

Reply:
good idea, I can add that as well.

Reply:
added